vision/detect/detect.go

```diff
@@ -14,22 +14,24 @@ import (
 	"io"
 	"os"
 
-	"cloud.google.com/go/vision"
+	"cloud.google.com/go/vision/apiv1"
```

Comment:
Correcting myself, NewImageFromURI and NewImageFromReader stay the same. goimports may pick the wrong package but renaming doesn't save us from this wrong picking. I also really don't have a good idea what a better name replacement should be. It is a part of a core doc, whatever we name it, it will end up at people's codebases given most users just copy/paste.